app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -251,6 +270,17 @@ private void navigateTo(Class<? extends GdgActivity> activityClass, Bundle addit
         mDrawerLayout.closeDrawers();
     }
 
+    protected void setNavDrawerItemSelected(String title) {
+        Menu menu = mNavigationView.getMenu();
```

Comment:
I used ids instead of titles in the develop branch to do this. It utilizes the `findItem()` function found in the Menu class. 

Looks like this
`        menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME)).setChecked(true);
`